### PR TITLE
LPS-181629 [Raylife Ap] Pointer on Policy Number should not be shown

### PR DESCRIPTION
**Pull request by eduardo-diniz**
[LPS-181629](https://issues.liferay.com/browse/LPS-181629)
[LPS-181567](https://issues.liferay.com/browse/LPS-181567)

### CODE REVIEW

**Comment by eduardo-diniz** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx [hunk 1]
```diff
@@ -568,7 +568,7 @@ const PoliciesTable = () => {
 					policiesList.push({
 						commission: `$${commissionValue.toFixed(2)}`,
 						externalReferenceCode,
-						isClickable: 'true',
+						isClickable: productName === 'Auto' ? 'true' : 'false',
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx:571]: It should be boolean ._.

**Comment by eduardo-diniz** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx (hunk 1 above)
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx]: I changed it but in other places we have this problem.

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx (hunk 1 above)
Comment (kevenleone) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx]: thanks sir!

**Pull request closed without merging**